### PR TITLE
add comments and change log level for meta

### PR DESCRIPTION
**Pull request by pengweisong**
2. remove some unused files

#### What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

#### What problem(s) does this PR solve?
Issue(s) number: 

Description:
1. add comments and adjust log level for meta
2. remove some unused files
   - the meta/processors/kv have been never used. After talked with @critical27 , we think could remove them.
   - the kv relative meta client code was also removed
   - the MetaCommon was also unused and removed

#### Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Comment by liwenhui-soul** [comment]
why do you remove some metaclient's code, could you make a short explanation?

**Comment by pengweisong** [comment]
@liwenhui-soul Have added some more explanation in the pr description.

**Review by critical27 — APPROVED** [review]
Good job, much more clear

**Comment by liwenhui-soul** [comment]
miss the MetaDataUpgrade.h and MetaDataUpgrade.cpp?

File: src/meta/http/MetaHttpDownloadHandler.h [hunk 1]
```diff
@@ -19,6 +19,12 @@ namespace meta {
 
 using nebula::HttpCode;
 
+/**
+ * @brief Download sst files from hdfs to every storaged download folder.
+ *        It will send download http request to every storaged, letting them
+ *        download the corressponding sst files.
+ *
+ */
 class MetaHttpDownloadHandler : public proxygen::RequestHandler {
```
[the remark this answers is not in the document]
Comment (pengweisong) [reply src/meta/http/MetaHttpDownloadHandler.h:28]: Most of them only have an process function, I think class-level comments are enough

[the remark this answers is not in the document]
Comment (liwenhui-soul) [reply src/meta/http/MetaHttpDownloadHandler.h:28]: for example, this class's function's name only figure that they are doing http operation, but not exactly describing what they are doing

[the remark this answers is not in the document]
Comment (pengweisong) [reply src/meta/http/MetaHttpDownloadHandler.h:28]: They are only functions inherited from RequestHandler, it is no use here.

**Review by liwenhui-soul — DISMISSED** [review]

File: src/meta/processors/BaseProcessor.h [hunk 2]
```diff
@@ -120,153 +123,379 @@ class BaseProcessor {
...
+   * @param spaceId
+   * @param name edge name
+   * @return ErrorOr<nebula::cpp2::ErrorCode, EdgeType>
    */
   ErrorOr<nebula::cpp2::ErrorCode, EdgeType> getEdgeType(GraphSpaceID spaceId,
                                                          const std::string& name);
 
   /**
-   * Fetch the latest version edge's schema.
+   * @brief Retch the latest version edge's schema by space id and tag id
```
[the remark this answers is not in the document]
Comment (pengweisong) [reply src/meta/processors/BaseProcessor.h]: I'll fix it

File: src/meta/processors/BaseProcessor.h [hunk 3]
```diff
@@ -120,153 +123,379 @@ class BaseProcessor {
...
+   * @tparam RESP
+   * @param hostKey
+   * @return SUCCEEDED means exist, E_NOT_FOUND means not, others means error.
+   */
   nebula::cpp2::ErrorCode hostExist(const std::string& hostKey);
 
   /**
-   * Check hosts has been include by zone or not.
-   * */
+   * @brief Check if any host in host list included by any zone.
```
[the remark this answers is not in the document]
Comment (pengweisong) [reply src/meta/processors/BaseProcessor.h:284]: I'll improve it

File: src/meta/processors/BaseProcessor.h [hunk 4]
```diff
@@ -120,153 +123,379 @@ class BaseProcessor {
...
+   * @tparam RESP
+   * @param start
+   * @param end
+   */
   void doRemoveRange(const std::string& start, const std::string& end);
 
   /**
-   * Scan keys from start to end, doesn't contain end.
-   * */
+   * @brief Range scan.
```
[the remark this answers is not in the document]
Comment (pengweisong) [reply src/meta/processors/BaseProcessor.h:190]: Good point, I'll add it.

File: src/meta/http/MetaHttpReplaceHostHandler.h [hunk 5]
```diff
@@ -17,6 +17,13 @@ namespace meta {
...
 using nebula::HttpCode;
 
+/**
+ * @brief It will replace host info in meta partition table from
+ *        backup host to current cluster host.
+ *        It should be called after ingesting meta sst files when
+ *        restore cluster.
+ *
+ */
 class MetaHttpReplaceHostHandler : public proxygen::RequestHandler {
```
[the remark this answers is not in the document]
Comment (pengweisong) [reply src/meta/http/MetaHttpReplaceHostHandler.h:30]: OK

File: src/meta/http/MetaHttpDownloadHandler.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (pengweisong) [reply src/meta/http/MetaHttpDownloadHandler.h:30]: Anyway, I'll add some comments.

File: src/meta/processors/config/SetConfigProcessor.h [hunk 6]
```diff
@@ -11,6 +11,10 @@
 namespace nebula {
 namespace meta {
 
+/**
+ * @brief Set configuration of given module's given name,
+ *
+ */
```
[the remark this answers is not in the document]
Comment (pengweisong) [reply src/meta/processors/config/SetConfigProcessor.h:17]: I'll add some comments.

File: src/meta/processors/service/ServiceProcessor.h [hunk 7]
```diff
@@ -11,6 +11,11 @@
 namespace nebula {
 namespace meta {
 
+/**
+ * @brief Sign in outer service info. It only used in listeners now, such as es
+ *        service info.
+ *
+ */
```
[the remark this answers is not in the document]
Comment (pengweisong) [reply src/meta/processors/service/ServiceProcessor.h:18]: OK, I'll add it.

**Review by liwenhui-soul — APPROVED** [review]

**Review by liwenhui-soul — APPROVED** [review]

**Review by critical27 — APPROVED** [review]

**Pull request merged**